viewer/src/main/java/rv/Globals.java

```diff
@@ -2,8 +2,17 @@
 
 import java.awt.Image;
 import java.io.IOException;
+
```

Comment:
Hm, are you sure you don't have Eclipse import sorting active accidentally? IIRC clang-format doesn't put empty lines between sections.

Reply:
fixed in aca3e21